mysql-test/suite/rpl/t/rpl_bug-lp-1667307.cnf

```diff
@@ -0,0 +1,30 @@
+!include ../my.cnf
+
+[mysqld.1]
+enforce-gtid-consistency=ON
+gtid-mode=ON
+master-info-repository=TABLE
```

Comment:
Strictly speaking, we don't need
`master-info-repository=TABLE`
`relay-log-info-repository=TABLE`
for the first server